cmd/frontend/graphqlbackend/observability.go

```diff
@@ -26,6 +27,11 @@ func init() {
 func (r *schemaResolver) TriggerObservabilityTestAlert(ctx context.Context, args *struct {
 	Level string
 }) (*EmptyResponse, error) {
+	// 🚨 SECURITY: Do not allow arbitrary users to set off alerts.
+	if err := backend.CheckCurrentUserIsSiteAdmin(ctx); err != nil {
```

Comment:
I think what you want is this:
```suggestion
	if err := backend.CheckUserIsSiteAdmin(ctx); err != nil {
```